### DISCUSSION TITLE
Feature Request: Import user's to Umbraco Id from other systems.

### DISCUSSION THREAD

**Discussion by dKumarmj** [umbraco/Umbraco.Cloud.Issues#795] [Ideas]
### Issue description

It would be nice if we have a  feature that would be to be able to import the back office users to Umbraco Id from other systems.

Feature in the sense, provides services, so developers can interact with Umbraco Id and they can create/modify the user's programmatically into the Umbraco Id from other systems like okta,pingFederate, Azure Ad etc...

**Comment by sajumb** [comment]
Hi @MjDhaneshKumar ,
Thanks for the feature request. It is much appreciated. I will let you know if we start working on this feature.
